content/doc/book/pipeline/syntax.adoc

```diff
@@ -274,7 +286,21 @@ dockerfile:: Execute the Pipeline, or stage, with a container built from a
 
 ////
 
+===== Additional Options
+
+These are a few options for two or more `agent` implementations. They are not
+required.
+
+customWorkspace:: A string. Run the Pipeline or individual `stage` this `agent`
+is applied to within this custom workspace, rather than the default. It can be
+either a relative path, in which case the custom workspace will be under the
+workspace root on the node, or an absolute path. This option is valid for
+`node`, `docker` and `dockerfile`.
```

Comment:
I think this would be clearer if the last line were separate: 
```
workspace root on the node, or an absolute path. 
+
This option is valid for `node`, `docker` and `dockerfile`.
```

What do you think?   If you agree, great, if not no big deal.